br/pkg/task/backup.go

```diff
@@ -184,11 +185,6 @@ func (cfg *BackupConfig) ParseFromFlags(flags *pflag.FlagSet) error {
 		cfg.UseCheckpoint = false
 		log.Info("since incremental backup is used, turn off checkpoint mode")
 	}
-	if cfg.UseBackupMetaV2 {
```

Comment:
we need to use backupmetaV2 feature to use less memory in BR, so it's better to use checkpoint feature both.